### ISSUE TITLE
Generate Time Series Movie with Monthly Steps

### ISSUE THREAD

**Issue by Christian-Mielke** [UBA-DE-Emissionsituation/space-emissions#12]
This might be a very nice feature for presenting the output data in an animated way.

**Comment by kevin-hausmann** [comment]
How do you like this @Christian-Mielke ?

![emissions](https://user-images.githubusercontent.com/5708074/117774926-41fed880-b23a-11eb-8d20-0f992201e1fe.gif)

**Comment by Christian-Mielke** [comment]
Yes, I really like this kind of animation. Well done!

**Comment by Christian-Mielke** [comment]
If we keep this grid layout maybe for the result collation we could potentially also integrate other data into this.

**Comment by kevin-hausmann** [comment]
Will keep this open, as this is only prototype right now.

**Comment by kevin-hausmann** [comment]
> 
> 
> If we keep this grid layout maybe for the result collation we could potentially also integrate other data into this.

Note: The grid used here comes from the TEMIS data (0.125°x0.125°) and can be very different for other emission estimation methods. However, the geopandas dataframe can certainly be augmented with additional info.

**Comment by Christian-Mielke** [comment]
Yes it is also pretty much different if you look at real world data from TROPOMI:

**Comment by Christian-Mielke** [comment]
![noxy](https://user-images.githubusercontent.com/79509063/120360587-34081900-c309-11eb-938e-1ac44b4cb51e.jpg)

**Comment by Christian-Mielke** [comment]
Obviously still needs masking of the garbage interpolation points though.

**Comment by kevin-hausmann** [comment]
![emissions](https://user-images.githubusercontent.com/5708074/144000667-0eee38b9-e6bd-4a9a-8e6e-44d7f045bd1c.gif)
